File: C/Program Menghitung Jumlah Huruf dan Angka.c
#include <stdio.h>
#include <ctype.h>

int main() {
    char karakter;
    int huruf_besar = 0, huruf_kecil = 0;
    int huruf_vokal = 0, huruf_konsonan = 0;
    int spasi = 0, angka = 0;

    printf("Program Menghitung Jumlah Huruf dan Angka\n");
    printf("visit us hobingoding.com\n\n");
    // Input Kalimat
    printf("Input Kalimat : ");
    while((karakter = getchar())!='\n') {
        // Hitung Huruf dan Angka
        huruf_konsonan++;
        if(karakter >= 'A' && karakter <= 'Z')
            huruf_besar++;
        if(karakter >= 'a' && karakter <= 'z')
            huruf_kecil++;
        if(toupper(karakter) == 'A' || toupper(karakter) == 'I' || toupper(karakter) == 'U' || toupper(karakter) == 'E' || toupper(karakter) == 'O')
            huruf_vokal++;
        if(karakter == ' ')
            spasi++;
        if(karakter >= '0' && karakter <= '9')
            angka++;
    }
    huruf_konsonan -= spasi + angka + huruf_vokal;

    printf("Hasil Penghitungan :\n");
    printf("Kapital : %d\n", huruf_besar);
    printf("Non-Kapital : %d\n", huruf_kecil);
    printf("Vokal : %d\n", huruf_vokal);
    printf("Konsonan : %d\n", huruf_konsonan);
    printf("Angka : %d\n", angka);
    printf("Spasi : %d\n", spasi);
    return 0;
}

File: Python/Program Menghitung Jumlah Huruf dan Angka.py
print("Program Menghitung Jumlah Huruf dan Angka")
print("visit us hobingoding.com")
print()
huruf_besar = 0
huruf_kecil = 0
huruf_vokal = 0
huruf_konsonan = 0
spasi = 0
angka = 0
# Input Kalimat
kalimat = input("Input Kalimat : ")
for i in range(0, len(kalimat)):
    # Hitung Huruf dan Angka
    huruf_konsonan += 1
    if(kalimat[i] >= 'A' and kalimat[i] <= 'Z'):
        huruf_besar += 1
    if(kalimat[i] >= 'a' and kalimat[i] <= 'z'):
            huruf_kecil += 1
    if(kalimat[i].upper() == 'A' or kalimat[i].upper() == 'I' or kalimat[i].upper() == 'U' or kalimat[i].upper() == 'E' or kalimat[i].upper() == 'O'):
            huruf_vokal += 1
    if(kalimat[i] == ' '):
            spasi += 1
    if(kalimat[i] >= '0' and kalimat[i] <= '9'):
            angka += 1
huruf_konsonan -= (spasi + angka + huruf_vokal)

print("Hasil Penghitungan :")
print("Kapital :", huruf_besar)
print("Non-Kapital :", huruf_kecil)
print("Vokal :", huruf_vokal)
print("Konsonan :", huruf_konsonan)
print("Angka :", angka)
print("Spasi :", spasi)
